<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="PE-HO-365" failures="1" tests="2" name="com.SecretariaTecnica.test.RevicionSTTest" time="112.874" errors="0" timestamp="01 oct. 2021 13:25:53 GMT" skipped="0">
  <testcase classname="com.SecretariaTecnica.test.RevicionSTTest" name="RevicionSTecnicaPortalFedepalma" time="59.831"/>
  <testcase classname="com.SecretariaTecnica.test.RevicionSTTest" name="RevicionSTecnicaPortalFedepalma1" time="53.043">
    <failure message="expected [El proceso se realizo correctamente.] but found [La compensacion N° 228 no se ha podido guardar: Exception of type &#039;CambioEstadosApi.BI.helper.FedepalmaException&#039; was thrown..]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [El proceso se realizo correctamente.] but found [La compensacion N° 228 no se ha podido guardar: Exception of type 'CambioEstadosApi.BI.helper.FedepalmaException' was thrown..]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.SecretariaTecnica.page.RevicionSTecnicaPage.ValidarGuardarRevicion(RevicionSTecnicaPage.java:109)
at com.SecretariaTecnica.test.RevicionSTTest.RevicionSTecnicaPortalFedepalma1(RevicionSTTest.java:55)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- RevicionSTecnicaPortalFedepalma1 -->
</testsuite> <!-- com.SecretariaTecnica.test.RevicionSTTest -->
